vendor/yubico/u2flib-server/.travis.yml

```diff
@@ -1,19 +1,21 @@
 language: php
 sudo: false
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
```

Comment:
5.6 is included via the matrix ;-)

It was basically run redundantly, so I removed one build.